common/lib/util/logger.ts

```diff
@@ -0,0 +1,94 @@
+import Platform from 'platform';
+
+enum LogLevels {
+    None = 0,
+    Error = 1,
+    Major = 2,
+    Minor = 3,
+    Micro = 4,
+}
+
+function pad(str: unknown, three?: number) {
```

Comment:
The abbreviation `str` is not helpful, nor is the name I assume is the expansion ('string'), especially with the type being `unknown`.

Reply:
I've rewritten this function, should be a lot more readable now.